### DISCUSSION TITLE
Is there a way to get firebase installation ID ?

### DISCUSSION THREAD

**Discussion by anilgr** [capawesome-team/capacitor-firebase#778] [Q&A] [answered]
I wanted to know about each installation of the app, which can be achieved by accessing the firebase installation ID. But, I don't see any plugin package with the name **capacitor-firebase/installations**. Is there any other way to get the installation ID, or do we need to add a new plugin here.

**Comment by robingenz** [comment, marked as answer]
Yes, we need another plugin for this. However, this should be done quickly. Feel free to [create a plugin request](https://github.com/capawesome-team/capacitor-firebase/issues/new/choose).

**Reply by anilgr** [reply]
Thanks. Created a plugin request  #779
